src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -67,15 +71,26 @@ public static SusiThought  scrapeQuora(String profile) {
 		JSONObject quoraProfile = new JSONObject();
 
 		Document userHTML = null;
-		String url = "https://www.quora.com/profile/" + profile;
+		String url = QUORA_URL_BASE + "/profile/" + profile;
 
 		try {
-			userHTML = Jsoup.connect(url).get();
```

Comment:
Have a simple question here, why need to change get() methods and to provide client agent? Will get() method be limited to scrape?